db/migrate/20160109042521_add_passing_game_logs_to_player.rb

```diff
@@ -0,0 +1,6 @@
+class AddPassingGameLogsToPlayer < ActiveRecord::Migration
```

Comment:
This migration is duplicating the association that is already stored in the player_id column on the passing_game_logs table. This migration (`AddPassingGameLogsToPlayer`) and its column ought to be rolled back.